### ISSUE TITLE
Updated Map

### ISSUE THREAD

**Issue by Havocfive** [DayZMod/DayZ#955]
Are there plans to make an updated in game map with the new POIs, roads, etc for 1.9.1?

**Comment by skigoggles** [comment]
if you mean the ones on TDR, then no. Would first need to find the authors, get permission and get agreement to add.

**Comment by Havocfive** [comment]
No, just the base 1.9.0 map with the DayZ POIs. Then have both available, in case a server operator wants to use the map without POIs.
